.github/actions/setup-localstack/action.yaml

```diff
@@ -18,11 +18,14 @@ runs:
         registry: ghcr.io
         username: ${{ inputs.username }}
         password: ${{ inputs.password }}
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.11'
     - shell: bash
       run: |
-            pip install localstack==2.3.1                                # Install LocalStack cli
+            pip install localstack==3.3.0                                # Install LocalStack cli
             # Below image was copied manually from localstack/localstack:2.2 and uploaded to ghcr
```

Comment:
Update the comment to reflect current process.

Reply:
Fixed.

Reply:
Did you push this change?